.github/labeler.yml

```diff
@@ -4,12 +4,16 @@
   - cmake/**
 'Component - Core':
   - Core/**
+  - Tests/UnitTests/Core/**
 'Component - Examples':
   - Examples/**
+  - Tests/UnitTests/Examples/**
 'Component - Fatras':
   - Fatras/**
+  - Tests/UnitTests/Fatras/**
 'Component - Plugins':
   - Plugins/**
+  - Tests/UnitTests/Plugins/**
```

Comment:
what about `'**/Plugins/**'` (and similar for others)? The `'` are needed if you start with `**`

Reply:
I don't have a strong opinion. I'm leaning towards being more explicit.